app/models/user.rb

```diff
@@ -55,6 +55,19 @@ def get_patient(id)
     end
   end
 
+  # Get multiple patients (that this user is allowed to get)
+  def get_patients(ids)
+    if has_role?(:enroller)
+      enrolled_patients.where(id: ids)
+    elsif has_role?(:public_health)
+      viewable_patients.where(id: ids)
+    elsif has_role?(:public_health_enroller)
+      viewable_patients.where(id: ids)
+    elsif has_role?(:admin)
+      Patient.where(id: ids)
```

Comment:
although this is not currently being used, is this the scope of patients we would like admins to be able to have access to? @holmesie